tests/apollo/test_skvbc_block_accumulation.py

```diff
@@ -0,0 +1,149 @@
+# Concord
+#
+# Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+
+import os.path
+import unittest
+import trio
+import random
+
+from util.bft import with_trio, with_bft_network, KEY_FILE_PREFIX, with_constant_load
+from util.skvbc_history_tracker import verify_linearizability
+from util import skvbc as kvbc
+
+import util.bft_network_partitioning as net
+import util.eliot_logging as log
+
+SKVBC_INIT_GRACE_TIME = 2
+BATCH_SIZE = 10
+NUM_OF_SEQ_WRITES = 1
+NUM_OF_PARALLEL_WRITES = 5
+MAX_CONCURRENCY = 4
```

Comment:
As you don't ensure that a number of written blocks equals to a specific value, you can run more transactions through more clients. Let's set NUM_OF_PARALLEL_WRITES = 100 and MAX_CONCURRENCY = 10

Reply:
Updated test script with modified configuration